docs/core/tools/dotnet-vstest.md

```diff
@@ -0,0 +1,127 @@
+---
+title: dotnet-vstest command | Microsoft Docs
+description: The dotnet-vstest command builds a project and all of its dependencies. 
+keywords: dotnet-vstest, CLI, CLI command, .NET Core
+author: guardrex
+ms.author: mairaw
+ms.date: 03/09/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 0e36c070-2242-41d3-96f1-aea0aca48d4d
+---
+
+# dotnet-vstest
+
+## Name
+
+`dotnet-vstest` - Runs tests from the specified files.
+
+## Synopsis
+
+`dotnet vstest [<TEST_FILE_NAMES>] [--Settings|/Settings] [--Tests|/Tests] [--TestAdapterPath|/TestAdapterPath] [--Platform|/Platform] [--Framework|/Framework] [--Parallel|/Parallel] [--TestCaseFilter|/TestCaseFilter] [--logger|/logger] [-lt|--ListTests|/lt|/ListTests] [--ParentProcessId|/ParentProcessId] [--Port|/Port] [--Diag|/Diag] [[--] <args>...]] [-?|--Help|/?|/Help]`
+
+## Description
+
+The `dotnet-vstest` command runs the `VSTest.Console` command-line application to run automated unit and coded UI application tests.
+
+## Arguments
+
+`TEST_FILE_NAMES`
+
+Run tests from the specified files. Separate multiple test file names by spaces.
```

Comment:
It might be more accurate to say "assemblies" rather than "files". This could be mis-interpreted as source files, as written.